README.md

```diff
@@ -1,42 +1,59 @@
-# About
-Current repository contains web interface for ostis platform ([here](https://github.com/ostis-ai) you can find all ostis modules).
+# SC-Web
+## About
+SC-Web is an intelligent knowledge base user interface. Currently, it is a part of [OSTIS Web Platform](https://github.com/ostis-ai/ostis-web-platform). 
+
+The long term idea of this project is to create a universal rendering mechanism for interfaces defined inside knowledge bases.
```

Comment:
I'll propose you what you can write this soon :)

Reply:
Leaving as-is for now. I agree that we should update this text when the project will become a bit more mature though!